ts/src/binance.ts

```diff
@@ -237,141 +237,137 @@ export default class binance extends Exchange {
                 'fees': 'https://www.binance.com/en/fee/schedule',
             },
             'api': {
-                // the API structure below will need 3-layer apidefs
                 'sapi': {
-                    // IP (sapi) request rate limit of 12 000 per minute
-                    // 1 IP (sapi) => cost = 0.1 => (1000 / (50 * 0.1)) * 60 = 12000
-                    // 10 IP (sapi) => cost = 1
-                    // UID (sapi) request rate limit of 180 000 per minute
-                    // 1 UID (sapi) => cost = 0.006667 => (1000 / (50 * 0.006667)) * 60 = 180000
                     'get': {
-                        // copy trading
-                        'copyTrading/futures/userStatus': 2,
-                        'copyTrading/futures/leadSymbol': 2,
-                        'system/status': 0.1,
-                        // these endpoints require this.apiKey
-                        'accountSnapshot': 240, // Weight(IP): 2400 => cost = 0.1 * 2400 = 240
-                        'account/info': 0.1,
-                        'margin/asset': 1, // Weight(IP): 10 => cost = 0.1 * 10 = 1
+                        'copyTrading/futures/userStatus': 20,
+                        'copyTrading/futures/leadSymbol': 20,
+                        'system/status': 1,
+                        'accountSnapshot': 2400,
+                        'account/info': 1,
+                        'margin/asset': 1,
                         'margin/pair': 1,
-                        'margin/allAssets': 0.1,
-                        'margin/allPairs': 0.1,
-                        'margin/priceIndex': 1,
-                        // these endpoints require this.apiKey + this.secret
-                        'spot/delist-schedule': 10,
-                        'asset/assetDividend': 1,
-                        'asset/dribblet': 0.1,
-                        'asset/transfer': 0.1,
-                        'asset/assetDetail': 0.1,
-                        'asset/tradeFee': 0.1,
-                        'asset/ledger-transfer/cloud-mining/queryByPage': 4.0002, // Weight(UID): 600 => cost = 0.006667 * 600 = 4.0002
+                        'margin/allAssets': 1,
+                        'margin/allPairs': 1,
+                        'margin/priceIndex': 10,
+                        'spot/delist-schedule': 100,
+                        'asset/assetDividend': 10,
+                        'asset/dribblet': 1,
+                        'asset/transfer': 1,
+                        'asset/assetDetail': 1,
+                        'asset/tradeFee': 1,
+                        'asset/ledger-transfer/cloud-mining/queryByPage': 600,
                         'asset/convert-transfer/queryByPage': 0.033335,
```

Comment:
such occasions are not a fault in parser, but these values are copied from existing binance.ts (as those endpoints are no longer found in docs , probably deprecated or removed), so we just retain them as-is